### PR TITLE
origin/olivia/UI bug fixes

### PR DESCRIPTION
**Pull request by oliviaxjiang**
### Description:
bug fixes (tackled four github issues) for better user experience

### Issues Fixed:
**Menu Spacing** [#61](#61)
added grey divider after each (except for the last) category as per the Figma design in Eatery Details

**Upcoming Menus - Grey Circle #74** [#74]
got rid of the grey circle for the dropdown button in Upcoming Menus

**Upcoming Menus - Drop Shadows #82** [#82]
changed the elevation of MenuCard drop shadow so it is less strong in Upcoming Menus

**Upcoming Menus - Drop Down on Whole Card** [#83]
made the whole MenuCard clickable to release the drop down information in Upcoming Menus

**also in upcoming menu viewmodel, got rid of the if conditional that resets the filters to default if all three filers are clicked (quality of life update ig)

### CODE REVIEW

File: app/src/main/java/com/cornellappdev/android/eateryblue/ui/screens/EateryDetailScreen.kt [hunk 1]
```diff
@@ -889,9 +889,13 @@ fun EateryMenuWidget(event: Event) {
                         }
                     }
                 }
+                if (categoryIndex != event.menu!!.lastIndex) {
+                    Divider(color = GrayZero,
+                        modifier = Modifier
+                            .height(10.dp)
+                    )
```
Comment (thisjustin123) [inline app/src/main/java/com/cornellappdev/android/eateryblue/ui/screens/EateryDetailScreen.kt:896]: There's some sus spacing here; make sure to run the formatter on your code before pushing:
image
Right click the code folder and hit "Reformat Code".

**Review by thisjustin123 — CHANGES_REQUESTED** [review]
Changes look good! But I think there's still a decent discrepancy on the eatery detail page. The items should be spaced with more padding between the lines of each item within a menu section--see Figma or iOS for what I mean there.
image

File: app/src/main/java/com/cornellappdev/android/eateryblue/ui/screens/EateryDetailScreen.kt (hunk 1 above)
Comment (oliviaxjiang) [reply app/src/main/java/com/cornellappdev/android/eateryblue/ui/screens/EateryDetailScreen.kt]: fixed! (first photo is before, second is after)
Screenshot 2024-02-28 at 2 46 04 PM
Screenshot 2024-02-28 at 5 28 17 PM

**Review by thisjustin123 — APPROVED** [review]
Great work! I just removed some random comment I found lying around since I won't make you wait around to remove one line but otherwise great stuff!!

🚀

**Pull request merged**